spec/bat/stemcell_spec.rb

```diff
@@ -88,6 +88,9 @@
       'bosh-warden-boshlite-ubuntu-trusty-go_agent' => false,
       'bosh-warden-boshlite-centos-go_agent' => false,
 
+      # Azure CPI does not support network reconfig
```

Comment:
You *did* implement it. `raise Bosh::Clouds::NotSupported` is a valid implementation. There is no need to change that implementation.